src/main/java/xyz/nucleoid/fantasy/mixin/registry/DimensionOptionsRegistryHolderMixin.java

```diff
@@ -13,8 +16,15 @@
 
 @Mixin(DimensionOptionsRegistryHolder.class)
 public class DimensionOptionsRegistryHolderMixin {
+    @Unique private static final Logger fantasy$LOGGER = LogUtils.getLogger();
+
     @ModifyArg(method = "method_45516", at = @At(value = "INVOKE", target = "Lcom/mojang/serialization/MapCodec;forGetter(Ljava/util/function/Function;)Lcom/mojang/serialization/codecs/RecordCodecBuilder;"))
     private static Function<Object, Registry<DimensionOptions>> fantasy$swapRegistryGetter(Function<Object, Registry<DimensionOptions>> getter) {
-        return (x) -> new FilteredRegistry<>(getter.apply(x), FantasyDimensionOptions.SAVE_PROPERTIES_PREDICATE);
+        try {
+            return (x) -> new FilteredRegistry<>(getter.apply(x), FantasyDimensionOptions.SAVE_PROPERTIES_PREDICATE);
+        } catch (Throwable e) {
```

Comment:
This will likely have little effect. For the try/catch to have an effect, this must be inside the lambda instead:
```java
(x) -> {
    try {...} catch(...) {...}
}
```